contracts/plugins/assets/curve/stakedao/StakeDAORecursiveCollateral.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import "../CurveRecursiveCollateral.sol";
+
+interface IStakeDAOVault is IERC20Metadata {
+    function token() external view returns (IERC20Metadata);
+
+    function liquidityGauge() external view returns (IStakeDAOGauge);
+}
+
+interface IStakeDAOGauge {
+    function claimer() external view returns (IStakeDAOClaimer);
+
+    function reward_count() external view returns (uint256);
+
+    function reward_tokens(uint256 index) external view returns (IERC20Metadata);
+}
+
+interface IStakeDAOClaimer {
```

Comment:
Which claimer are you looking at? The one that StakeDAO currently uses on their website has two parameters, `guages` and `claimVeSDT`. [Contract](https://etherscan.io/address/0x539e65190a371ce73244a98dec42ba635cca512c).

Reply:
the arg should be `false` yes?